clang/lib/Headers/mm_malloc.h

```diff
@@ -13,7 +13,7 @@
 #include <stdlib.h>
 
 #ifdef _WIN32
-#include <malloc.h>
+#include <corecrt_malloc.h>
```

Comment:
mingw-w64 doesn't have `corecrt_malloc.h` (yet), so unfortunately we can't go this way in general. Perhaps we could for `!defined(__MINGW32__)` or `defined(_MSC_VER)` though.

Also, typo in the commit message `mordularising`.

Reply:
Thanks for the typo correction!

I think that I can add in a `!defined(__MINGW32__)` condition.